src/main/java/lab3/ui/terminalConsole.java

```diff
@@ -0,0 +1,104 @@
+package lab3.ui;
+
+import lab3.game.Board;
+import lab3.game.Coordinates;
+
+import java.util.Scanner;
+
+import static lab3.game.Board.currentPlayerSymbol;
+import static lab3.game.winRecord.callScore;
+
+
+
+public class terminalConsole {
+
+    private final Board board;
+
+    /**
+     * Creates a terminalConsole instance to interact with the board object
+     * @param board the tic tac toe board
+     */
+    public terminalConsole(Board board) {
+        this.board = board;
+    }
+
+    /**
+     * Displays the current board state
+     */
+    public void displayBoard() {
+        System.out.println(board);
+    }
+
+    /**
+     * Prints a message to the console
+     * @param message the message to be printed
+     */
+    public static void println(String message) {
+        System.out.println(message);
+    }
+
+    /**
+     * Displays a message if a player has reached a win state
+     */
+    public static void displayWin() {
+        println("PLAYER " + currentPlayerSymbol() + " HAS WON");
+        println(String.format("THE SCORE IS NOW %s (X) VS %s (O)", callScore().XWins(), callScore().OWins()));
+    }
+
+    /**
+     * Displays a tie message if neither player was able to reach a win state
+     */
+    public static void displayTie() {
+        println("TIE GAME, NO ONE WINS!");
+        println(String.format("THE SCORE REMAINS %s (X) VS %s (O)", callScore().XWins(), callScore().OWins()));
+    }
```

Comment:
Using a the `callScore` method here is really just a sneaky way to use a global variable (your `winRecord.gameScore` variable).  These functions would be better as pure functions that take a `winRecord` object, something like so:

    public static void displayWin(char winnerSymbol, winRecord score) {
        println("PLAYER " + winnerSymbol + " HAS WON");
        println(String.format(" ... %s .. %s .. ", score.XWins(), score.OWins());
    }